### DISCUSSION TITLE
Testing a glsl shader

### DISCUSSION THREAD

**Discussion by ggarra13** [AcademySoftwareFoundation/OpenRV#74] [General]
I am trying to test a glsl shader (I am starting with one that ships with OpenRV).

I set my support path to the .gto and .glsl files.

export RV_SUPPORT_PATH=$PWD/OpenRV/src/plugins/rv-packages/lat_long_viewer

The documentation at:
https://knowledge.autodesk.com/support/shotgrid/learn-explore/caas/CloudHelp/cloudhelp/ENU/SG-RV/files/rv-manuals/rv-reference-manual/SG-RV-rv-manuals-rv-reference-manual-chapter-three-html-html.html

According to the docs, I need to open a new session/view in the session manager but I cannot make it work.  Is this supported in OpenRV?  The dpcuentation at:

https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_RV_rv_manuals_rv_user_manual_rv_user_manual_chapter_five_html#5-the-session-and-the-session-manager

shows pictures of the session manager, but they look nothing like what's in OpenRV.

**Comment by compaga** [comment]
Hi ggarra13,

Although Autodesk RV documentation might still be relevant. the Open RV document is currently available in the repo.
https://github.com/AcademySoftwareFoundation/OpenRV/blob/main/doc/rv-manuals/rv-reference-manual/rv-reference-manual-chapter-three.md 

I know it is likely the same but I simply wanted to let you know.
Alain

**Comment by ggarra13** [comment]
But is the Session Manager shipping with OpenRV?  The manual says it is supposed to be there, but there appears no way to access it.  I can list the Sequence, Tracks and individual image, but nothing more.

**Comment by compaga** [comment]
Bottom left icon will expand the session manager. 
![Screenshot 2023-01-27 at 10 38 47 AM](https://user-images.githubusercontent.com/24920866/215126191-8526c102-e2af-4075-9a81-c4e6debdc11f.png)

**Comment by savoiei** [comment]
Hi ggarra13, 

The Session Manager should be accessible through the "Tools / Session Manager" menu, "X" hotkey or through the button at the bottom-left of RV's main UI. It should be enabled be default, but if it is still not accessible, you might want to have a look in the "OpenRV / Preferences / Packages" menu and see if the package is installed and loaded. You will have to click on the "Show Hidden Packages" at the bottom-right of the Preferences UI to see it in the Packages list. I will come back to you for the question about glsl shader.

Thanks!
Ian

**Comment by ggarra13** [comment]
Ok. I got the GLSL shader to work (lat_long_viewer).  However I cannot get a UI to set its parameters nor the Image menu that the bundled documentation says it should have.
